otx/algorithms/detection/adapters/mmdet/models/backbones/mmov_backbone.py

```diff
@@ -1,24 +1,29 @@
-# Copyright (C) 2022 Intel Corporation
+"""Backbone Class of OMZ model for mmdetection backbones."""
+# Copyright (C) 2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
 
 from mmdet.models.builder import BACKBONES
 
-from ...mmov_model import MMOVModel
+from otx.mpa.modules.ov.models.mmov_model import MMOVModel
```

Comment:
Should otx.mpa.modules.ov.models.mmov_model go to common folder?

Reply:
I agree, but not in that PR :) because I'd like to change common as well, which would make the change very large.